lib/common/grpc-service.js

```diff
@@ -332,24 +364,20 @@ GrpcService.convertValue_ = function(value) {
     convertedValue = {
       structValue: GrpcService.objToStruct_(value)
     };
-  } else if (is.date(value)) {
```

Comment:
A `Struct` protobuf message doesn't support storing a Date, so I don't think we can keep it.

And which value should we convert to a Date object?